modular_skyrat/modules/modular_implants/code/misc_devices.dm

```diff
@@ -99,3 +99,68 @@
 	if(!uses)
 		qdel(src)
 
+/obj/item/nif_hud_adapter
+	name = "NIF HUD retrofitter"
+	desc = "A kit that modifies select glasses to display HUDs for NIFs"
+	icon = 'modular_skyrat/master_files/icons/donator/obj/kits.dmi'
+	icon_state = "partskit"
+
+	/// Can this item be used multiple times? If not, it will delete itself after being used.
+	var/multiple_uses = FALSE
+	/// List containing all of the glasses that we want to work with this.
+	var/static/list/glasses_whitelist = list(
+		/obj/item/clothing/glasses/trickblindfold,
+		/obj/item/clothing/glasses/monocle,
+		/obj/item/clothing/glasses/fake_sunglasses,
+		/obj/item/clothing/glasses/regular,
+		/obj/item/clothing/glasses/eyepatch,
+		/obj/item/clothing/glasses/osi,
+		/obj/item/clothing/glasses/phantom,
+		/obj/item/clothing/glasses/salesman, // Now's your chance.
+		/obj/item/clothing/glasses/nice_goggles,
+		/obj/item/clothing/glasses/thin,
+		/obj/item/clothing/glasses/biker,
+		/obj/item/clothing/glasses/sunglasses/gar,
+	)
+
+/obj/item/nif_hud_adapter/examine(mob/user)
+	. = ..()
+	var/list/compatible_glasses_names = list()
+	for(var/obj/item/glasses_type as anything in glasses_whitelist)
+		var/glasses_name = initial(glasses_type.name)
+		if(!glasses_name)
+			continue
+
+		compatible_glasses_names += glasses_name
+
+	if(length(compatible_glasses_names))
+		. += span_cyan("This item will work on the following glasses: [english_list(compatible_glasses_names)].")
```

Comment:
how does this look ingame

Reply:
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/68373373/5917e5b9-9384-4d6b-a0c3-0bdb5da5fb8a)

Reply:
Does it need to be bold, though? I feel like that's a bit intense for something like this.
Also I'd add an empty line between the rest of the description and this portion.

Reply:
The wasn't made bold intentionally, this is just something that happens with `span_cyan()`

Reply:
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/68373373/fff37b6b-39a2-4789-8a79-2cc364a2b679)